docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-bootstrap-switches-ref.adoc

```diff
@@ -0,0 +1,246 @@
+|===
+
+ifdef::BA[]
+.Callback Bootstrap Switches
+[cols="2,1a,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.jbpm.ht.callback`
+|`mvel`
+
+`ldap`
+
+`db`
+
+`jaas`
+
+`props`
+
+`custom`
+|`jaas`
+|Specifies the implementation of user group callback to be used:
+
+* `mvel`: Default; mostly used for testing.
+* `ldap`: LDAP; requires additional configuration in the <<jbpm.usergroup.callback.properties, `jbpm.usergroup.callback.properties`>> file.
+* `db`: Database; requires additional configuration in the <<jbpm.usergroup.callback.properties, `jbpm.usergroup.callback.properties`>> file.
+* `jaas`: JAAS; delegates to the container to fetch information about user data.
+* `props`: A simple property file; requires additional file that will keep all information (users and groups).
+* `custom`: A custom implementation; specify the fully qualified name of the class in the `org.jbpm.ht.custom.callback` property.
+
+|`org.jbpm.ht.custom.callback`
+|Fully qualified name
+|N/A
+|A custom implementation of the `UserGroupCallback` interface in case the `org.jbpm.ht.callback` property is set to `custom`.
+|===
+endif::BA[]
+
+.Other Bootstrap Switches
+[cols="2,1,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.maven.settings.custom`
+|Path
+|N/A
+|The location of a custom `settings.xml` file for Maven configuration.
+
+|`kie.server.jms.queues.response`
+|String
+|`queue/KIE.SERVER.RESPONSE`
+|The response queue JNDI name for JMS.
+
+|`org.drools.server.filter.classes`
+|`true`, `false`
+|`false`
+|When set to `true`, the Drools {KIE_SERVER} extension accepts custom classes annotated by the `XmlRootElement` or `Remotable` annotations only.
+
+ifdef::BA[]
+|`org.kie.server.bypass.auth.user`
+|`true`, `false`
+|`false`
+|Allows to bypass the authenticated user for task-related operations, for example queries.
+endif::BA[]
+
+|`org.kie.server.domain`
+|String
+|N/A
+|The JAAS `LoginContext` domain used to authenticate users when using JMS.
+
+|`org.kie.server.repo`
+|Path
+|`.`
+|The location where {KIE_SERVER} state files will be stored.
+
+|`org.kie.server.sync.deploy`
+|`true`, `false`
+|`false`
+|Instructs the {KIE_SERVER} to hold the deployment until the controller provides the containers deployment configuration. This property affects only the servers running in managed mode. The options are as follows:
+
+* `false`; the connection to the controller is asynchronous. The application starts, connects to the controller, and once successful, deploys the containers. The application accepts requests even before the containers are available.
+* `true`; the deployment of the server application joins the controller connection thread with the main deployment and awaits its completion. This option can lead to a potential deadlock in case more applications are on the same server instance. It is strongly recommended to use only one application (the server) on one server instance.
```

Comment:
Instead of " It is strongly recommended to use only one application (the server) on one server instance...."
Can we say "Red Hat recommends....." OR "You should use only one....."

Reply:
"It is recommended that you use only one application (the server) on one server instance." I don't think "should" is instructive enough.